corehq/warehouse/loaders/application.py

```diff
@@ -0,0 +1,53 @@
+from corehq.apps.app_manager.models import Application
+from corehq.warehouse.const import APPLICATION_STAGING_SLUG, APPLICATION_DIM_SLUG
+from corehq.warehouse.dbaccessors import get_application_ids_by_last_modified
+from corehq.warehouse.etl import HQToWarehouseETLMixin, CustomSQLETLMixin
+from corehq.warehouse.loaders.base import BaseStagingLoader, BaseLoader
+from corehq.warehouse.models import ApplicationStagingTable, ApplicationDim
+from dimagi.utils.couch.database import iter_docs
+
+
+class ApplicationStagingLoader(HQToWarehouseETLMixin, BaseStagingLoader):
+    """
+    Represents the staging table to dump data before loading into the ApplicationDim
+
+    Grain: application_id
```

Comment:
for my understanding, what is `Grain`?

Reply:
it's the level of the aggregation ie. this is one row per `application_id`